lib/Models/CsvCatalogItem.js

```diff
@@ -192,6 +193,20 @@ defineProperties(CsvCatalogItem.prototype, {
         get : function() {
             return CsvCatalogItem.defaultPropertiesForSharing;
         }
+    },
+
+    /**
+     * Gets or sets the tableStyle object
+     * TODO: add definition for subfields
+     * @type {Object}
+     */
+    tableStyle :  {
```

Comment:
It actually does get used and it's the only way I could get tableStyle to get handled properly across load, setting, and sharing.